rebasehelper/spec_hooks/replace_old_version.py

```diff
@@ -20,12 +20,23 @@
 # Authors: Petr Hracek <phracek@redhat.com>
 #          Tomas Hozza <thozza@redhat.com>
 
+import urllib.parse
```

Comment:
Does this work with both python2 and python3?

Reply:
It didn't work for python2, should be fixed now